app/src/main/res/values/strings.xml

```diff
@@ -411,4 +410,10 @@
     <string name="pref_theme_muted_purple">Muted Purple</string>
     <string name="no_network_permissions">Jellyfin requires network permissions to function</string>
     <string name="press_to_close">Press to close</string>
+    <string name="lbl_special">Special</string>
+    <string name="lbl_season_number">S%1$d</string>
+    <string name="lbl_episode_number">E%1$d</string>
+    <string name="lbl_next_up_extended">Enabled, extended</string>
+    <string name="lbl_next_up_minimal">Enabled, minimal</string>
```

Comment:
We might want to use different names to better explain the difference between extended and minimal. I'm not sure how to name it though.

Reply:
Pushed a commit:

Show Next Up screen
- With thumbnail
- Without thumbnail
- Never

Alternatively, still using the 'Enable' wording:

Enable Next Up screen
- Extended (with thumbnail)
- Minimal (without thumbnail)
- Disabled